test/models/ConferenceTest.scala

```diff
@@ -0,0 +1,151 @@
+      // lists
+      assert(Conference.convertMarkdownToHTML("- "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("+ "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("* "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("-"+testingString) == "<p>-"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("+"+testingString) == "<p>+"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("*"+testingString) == "<p>*"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("- "+testingString+"\n- "+testingString)
+        == "<ul>\n<li>"+testingString+"</li>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("1. "+testingString) == "<ol>\n<li>"+testingString+"</li>\n</ol>\n")
+      assert(Conference.convertMarkdownToHTML("1."+testingString) == "<p>1."+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("2)"+testingString) == "<p>2)"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("1. "+testingString+"\n2. "+testingString)
+        == "<ol>\n<li>"+testingString+"</li>\n<li>"+testingString+"</li>\n</ol>\n")
+      // links
+      val urlTestingString : String = "http:"+testingString
+      assert(Conference.convertMarkdownToHTML("<"+urlTestingString+">")
+        == "<p><a href=\""+urlTestingString+"\">"+urlTestingString+"</a></p>\n")
+      assert(Conference.convertMarkdownToHTML("["+testingString+"]"+"("+urlTestingString+")")
+        == "<p><a href=\""+urlTestingString+"\">"+testingString+"</a></p>\n")
+      // code
+      assert(Conference.convertMarkdownToHTML("`"+testingString+"`") == "<p><code>"+testingString+"</code></p>\n")
+      assert(Conference.convertMarkdownToHTML("    "+testingString) == "<pre><code>"+testingString+"\n</code></pre>\n")
+      assert(Conference.convertMarkdownToHTML("```\n"+testingString+"\n```") == "<pre><code>"+testingString+"\n</code></pre>\n")
+    }
+  }
+
+  @Test
+  def testGetInfoAsHTML() : Unit = {
+    /*
+     * Mostly a placeholder for potential future changes, which need to be tested differently.
+     */
+    val r : Random = new Random()
+    var testingInfo : String = ""
+    var testingConf : Conference = null
+    for (i <- 0 to 10000) {
+      testingInfo = ConferenceTest.generateRandomString(r.nextInt(200))
+      testingConf = Conference(Some("uuid"), Some("someconf"), Some("XX"), Some("G"), Some("X"),
+        None, None, Some(false), Some(true), Some(false), Some(true), None, None, None, info = Some(testingInfo))
+      assert(testingConf.getInfoAsHTML() == Conference.convertMarkdownToHTML(testingInfo))
+    }
+    // test nulls throwing a NullPointerException
```

Comment:
But should they. I know they do however should they really.  Are you not, you trying to avoid it?

Reply:
Sure, I guess you mean the XSS protection is still missing?

Reply:
Let me ask differently. where would the np happen and how could it be avoided?